packages/genesys-spark-components/src/components/stable/gux-table/gux-table.light.scss

```diff
@@ -101,6 +101,12 @@ gux-table {
 
```

Comment:
This could be simplified a small bit more to
```
tr {
  background-color: var(--gse-ui-dataTableItems-cell-defaultBackgroundColor);

  &:hover {
    background-color: var(--gse-ui-dataTableItems-cell-hoverBackgroundColor);
  }

  &:nth-child(2n) {
    background-color: var(--gse-ui-dataTableItems-cell-altBackgroundColor);
  
    &:hover {
      background-color: var(--gse-ui-dataTableItems-cell-hoverBackgroundColor);
    }
  }
}
```
The second hover block does look odd/redundant so we might have to add a comment to stop it getting removed in the future.